code/game/objects/items/devices/geiger.dm

```diff
@@ -2,9 +2,8 @@
 
 /obj/item/device/geiger_counter
 	name = "geiger counter"
-	desc = "a device about the size of a briefcase, used for detecting and measuring ambient radiation."
+	desc = "A device used for detecting and measuring ambient radiation."
 	icon_state = "geiger_counter"
-	w_class = W_CLASS_LARGE
```

Comment:
I think the default w_class is medium, but these should probably be W_CLASS_SMALL so they're pocket sized like multitools and flashlights

Reply:
done